consensus/istanbul/backend/backend.go

```diff
@@ -316,3 +352,64 @@ func (sb *backend) HasBadProposal(hash common.Hash) bool {
 func (sb *backend) Close() error {
 	return nil
 }
+
+// IsQBFTConsensus returns whether qbft consensus should be used
+func (sb *backend) IsQBFTConsensus() bool {
+	// If qbftBlock is not defined in genesis, then use legacy ibft
+	if sb.qbftConsensusEnabled {
+		return true
+	}
+	if sb.chain != nil {
+		return sb.IsQBFTConsensusForHeader(sb.chain.CurrentHeader())
+	}
+	return false
+}
+
+// IsQBFTConsensusForHeader checks if qbft consensus is enabled for the block height identified by the given header
+func (sb *backend) IsQBFTConsensusForHeader(header *types.Header) bool {
+	// If qbftBlock is not defined in genesis, then use legacy ibft
```

Comment:
invalid comment, copy-paste error by the looks of it

Reply:
Updated the comment